packages/test/test-gc-sweep-tests/src/testDataObjects.ts

```diff
@@ -25,3 +51,94 @@ export class TestDataObject extends DataObject {
         return this.context;
     }
 }
+
+export class DataObjectManyDDSes extends BaseTestDataObject {
+    public static get type(): string {
+        return "TestDataObjectWithEveryDDS";
+    }
+
+    private readonly opHandlers: OpHandler[] = [];
+    private readonly handleOpHandlers: HandleOpHandler[] = [];
+    private readonly randomOpHandlers: IRandomOpHandler[] = [];
+    private random: Random | undefined;
+
+    public setRandom(random: Random) {
+        this.random = random;
+        for (const opHandler of this.opHandlers) {
+            opHandler.random = random;
+        }
+    }
+
+    public async generateAddHandleOp(handle: IFluidHandle): Promise<IFluidHandle | undefined> {
+        assert(this.random !== undefined, `Random for TestDataObjectWithEveryDDS needs to be set!`);
+        const handler: HandleOpHandler = this.random.pick(this.handleOpHandlers);
+        return handler.executeRandomAddHandleOp(handle);
+    }
+
+    public async generateRemoveHandleOp(): Promise<IFluidHandle | undefined> {
+        assert(this.random !== undefined, `Random for TestDataObjectWithEveryDDS needs to be set!`);
+        const handler: HandleOpHandler = this.random.pick(this.handleOpHandlers);
+        return handler.executeRandomRemoveHandleOp();
+    }
+
+    protected async hasInitialized(): Promise<void> {
+        const consensusQueue: ConsensusQueue<IFluidHandle> = ConsensusQueue.create(this.runtime);
+        const consensusQueueHandler = new ConsensusQueueHandler(consensusQueue);
+        this.opHandlers.push(consensusQueueHandler);
+        this.handleOpHandlers.push(consensusQueueHandler);
+
+        const consensusRegisterCollection: ConsensusRegisterCollection<IFluidHandle> =
+            ConsensusRegisterCollection.create(this.runtime);
+        const consensusRegisterCollectionHandler = new ConsensusRegisterCollectionHandler(consensusRegisterCollection);
+        this.opHandlers.push(consensusRegisterCollectionHandler);
+        this.handleOpHandlers.push(consensusRegisterCollectionHandler);
+
+        const ink: Ink = Ink.create(this.runtime);
+        const inkHandler = new InkHandler(ink);
+        this.opHandlers.push(inkHandler);
+        this.randomOpHandlers.push(inkHandler);
+
+        const sharedCell: SharedCell<IFluidHandle> = SharedCell.create(this.runtime);
+        const sharedCellHandler = new SharedCellHandler(sharedCell);
+        this.opHandlers.push(sharedCellHandler);
+        this.handleOpHandlers.push(sharedCellHandler);
+        this.randomOpHandlers.push(inkHandler);
```

Comment:
>inkHandler [](http://example.com/codeflow?start=35&length=10)

inkHandler or sharedCellHandler ? Same as the other 2 below ?

Reply:
Woops, I'll need to fix this.